app/forms/password_form.rb

```diff
@@ -10,10 +10,12 @@ def initialize(user)
   end
 
   def submit(params)
-    self.password = params[:password]
+    submitted_password = params[:password]
+
+    self.password = submitted_password
 
     if valid? && user_valid?
-      @user.password = params[:password]
+      @user.password = submitted_password
```

Comment:
Do you actually prefer this code?  I looks worse to me.  Can/should we instead disable the warning?